code/src/pages/products/ProductsPage.tsx

```diff
@@ -29,7 +51,7 @@ const ProductsPage = () => {
         />
         <AppBox className="spa-products-page__info">
           <AppTypography className="spa-products-page__count" component="span">
-            {products?.length || 0}
+            {productsData?.itemsCount}
```

Comment:
Maybe it would be better to move above the return statement and if there are no product items set 0. What do you think?

Reply:
Thanks, I moved it above